Cargo.toml

```
@@ -24,7 +24,8 @@ assets = [
     ["target/release/rad-untrack", "usr/bin/rad-untrack", "755"],
     ["target/release/rad-remote", "usr/bin/rad-remote", "755"],
     ["target/release/git-remote-rad", "usr/bin/git-remote-rad", "755"],
-    ["radicle-tools.1.gz", "usr/share/man/man1/radicle-tools.1.gz", "644"],
+    ["rad.1.gz", "usr/share/man/man1/rad.1.gz", "644"],
+    ["checkout/rad-checkout.1.gz", "usr/share/man/man1/rad-checkout.1.gz", "644"]
```

The compiled man pages are located in the top-level directory:
```suggestion
    ["rad-checkout.1.gz", "usr/share/man/man1/rad-checkout.1.gz", "644"]
```